docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx

```diff
@@ -1,172 +1,339 @@
 
 ```swift
-struct UserCredentials {
-    let user: User
-    let token: UserToken
-}
+let user = User(
+    id: userId,
+    name: "Martin"
+)
+```
 
-extension UserCredentials {
-    static let demoUser = UserCredentials(
-        user: User(
-            id: "testuser",
-            name: "Test User",
-            imageURL: URL(string: "https://vignette.wikia.nocookie.net/starwars/images/2/20/LukeTLJ.jpg")!,
-            customData: [:]
-        ),
-        token: UserToken(rawValue: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdHJlYW0tdmlkZW8tZ29AdjAuMS4wIiwic3ViIjoidXNlci90ZXN0dXNlciIsImlhdCI6MTY2NjY5ODczMSwidXNlcl9pZCI6InRlc3R1c2VyIn0.h4lnaF6OFYaNPjeK8uFkKirR5kHtj1vAKuipq3A5nM0")
-    )
-}
+**Initialize the Stream Client**. Next we initialize the client by passing the API Key, user and user token.
+
+```swift
+self.client = StreamVideo(
+    apiKey: apiKey,
+    user: user,
+    token: .init(stringLiteral: token)
+)
+```
+
+**Create and join call** After the user and client are created, we create a call like this:
+
+```swift
+self.call = client.call(callType: "audio_room", callId: callId)
+
+try await call.join(create: true)
+```
+
+As soon as you use call.join the connection for video & audio is setup.
+
+Lastly, the UI is rendered by observing call.state (participants and connection states):
+
+```swift
+call.state.participantCount
+call.state.reconnectionStatus
+```
+
+You'll find all relevant state for the call in `call.state` and `call.state.participants`. The documentation on [Call state and Participant state](../../guides/call-and-participant-state) explains this in further detail.
+
+### Step 4 - Joining from the web
+
+To make this a little more interactive, let's join the call from your browser.
+
+<TokenSnippet sampleApp='meeting' displayStyle='join' />
+
+On your iOS device, you'll see the text update to 2 participants. Let's keep the browser tab open as you go through the tutorial.
+
+### Step 5 - Rendering Video
+
+In this next step we're going to:
+
+- Render your local & remote participant video.
+
+Let's update the body of our VideoCallApp View with following code.
+
+```swift 
+var body: some Scene {
+        WindowGroup {
+            VStack {
+                if callCreated {
+                    GeometryReader { proxy in
+                        ZStack {
+                            if let participant = call.state.remoteParticipants.first {
+                                VideoRendererView(id: participant.id, size: proxy.size) { videoRenderer in
+                                    videoRenderer.handleViewRendering(for: participant, onTrackSizeUpdate: { _, _ in })
+                                }
+                                .ignoresSafeArea()
+                            } else {
+                                Color.black
+                            }
+
+                            if let me = call.state.me {
+                                VStack {
```

Comment:
On one of the screenshots, this didn't respect the safe area (the other one was ok).